homeassistant/components/sentry/__init__.py

```diff
@@ -120,19 +122,19 @@ def get_channel(version: str) -> str:
 
 def process_before_send(
     hass: HomeAssistant,
-    options,
+    options: MappingProxyType[str, Any],
     channel: str,
     huuid: str,
     system_info: dict[str, bool | str],
     custom_components: dict[str, Integration],
-    event,
-    hint,
+    event: dict[str, Any],
+    hint: dict[str, Any],
 ):
     """Process a Sentry event before sending it to Sentry."""
     # Filter out handled events by default
     if (
         "tags" in event
-        and event.tags.get("handled", "no") == "yes"
+        and event.tags.get("handled", "no") == "yes"  # type: ignore[attr-defined]
```

Comment:
So `event` is not actually a `dict`?

Reply:
It's actually `dict[str, Any]` defined in `sentry_sdk._types`.

I can find anywhere where they set it as attribute, it's always used as a regular key:
https://github.com/getsentry/sentry-python/search?q=event.tags

Reply:
Can you update it? 🙂

Reply:
Eventually, I will, please have patience. Thanks 👍

Reply:
Adjusted